src/mixins/nft.js

```diff
@@ -352,6 +355,11 @@ export default {
           })
         : [defaultEdition];
     },
+    nftBookAvailablePriceLabel() {
+      const purchasePrice = this.nftEditions.find(item => item.stock > 0)
```

Comment:
Should refactor to avoid using `defaultEdition` to put listing data in the future, confusing since `this.nftEditions` should not exists out side book store API